data-prepper-plugins/s3-enricher-processor/build.gradle

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+dependencies {
+    implementation project(path: ':data-prepper-plugins:common')
+    implementation project(':data-prepper-plugins:aws-plugin-api')
+    implementation 'software.amazon.awssdk:sdk-core'
+    implementation 'software.amazon.awssdk:sts'
+    implementation 'io.micrometer:micrometer-core'
+    implementation 'org.json:json'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+    implementation 'org.projectlombok:lombok:1.18.22'
+    implementation libs.parquet.common
+    implementation 'dev.failsafe:failsafe:3.3.2'
```

Comment:
What is this? Are we using it?

Reply:
This is used in the S3InputStream which is a dependent in S3InputFile. Once the S3 common is extracted into a separate module we don't need it anymore in this project.